index.js

```diff
@@ -11,7 +11,7 @@ const loadProbot = (appFn) => {
     new Probot({
       id: process.env.APP_ID,
       secret: process.env.WEBHOOK_SECRET,
-      privateKey: findPrivateKey(),
+      privateKey: getPrivateKey(),
```

Comment:
how would you usually configure the private key on lambda? I assume you wouldn't upload a `*.pem` file, so the only option really is to define the `PRIVATE_KEY` option? I think I'd remove the `getPrivateKey` method altogether and instead set

```suggestion
      privateKey: process.env.PRIVATE_KEY,
```

Reply:
Given that `getPrivateKey()` does return `process.env.PRIVATE_KEY` as one of it's options and in the README there is a bit about using a `.pem`, I figured it would be safer to use this new package. But you're right, whenever I use lambda with serverless, I use the `serverless-dotenv-plugin` with `.env` files.

I can't say for a fact it is impossible to use a `*.pem` so I thought this would give a bit of flexibility. As things stand, this package is unusable with the latest `@probot` which is the reason for this PR.

Reply:
> As things stand, this package is unusable with the latest `@probot` which is the reason for this PR.

I agree, let's merge this PR and make a follow up PR for further discussion